python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -315,18 +320,21 @@ def _concat_treelite_models(self):
         to create a single model. The concatenated model is then converted to
         bytes format.
         """
-
         mod_bytes = []
+        model_protobuf_futures = list()
         for w in self.workers:
-            mod_bytes.append(self.rfs[w].result().model_pbuf_bytes)
+            model_protobuf_futures.append(
+                dask.delayed(RandomForestRegressor._get_protobuf_bytes)
+                (self.rfs[w]))
+        mod_bytes = self.client.compute(model_protobuf_futures, sync=True)
         last_worker = w
         all_tl_mod_handles = []
         model = self.rfs[last_worker].result()
         for n in range(len(self.workers)):
             all_tl_mod_handles.append(model._tl_model_handles(mod_bytes[n]))
-        concat_model_handle = model._concatenate_treelite_handle(
+
+        model._concatenate_treelite_handle(
             treelite_handle=all_tl_mod_handles)
-        model._concatenate_model_bytes(concat_model_handle)
```

Comment:
Do we still need this if we are not keeping the result anywhere? Or should this be stored somewhere?

Reply:
this function updates the `self.model_pbuf_bytes` variable for `model` and then `model` is used to set the `self.local_model`. The `self.local_model` is the RF model used to run lazy predict/ gpu predict in dask RF.